jobs/merge-branches.groovy

```diff
@@ -79,15 +94,58 @@ def checkArgs() {
 }
 
 
-def getGaeVersionName() {
+String getGaeVersionName() {
```

Comment:
This may be a good idea, but it's really different from all the other jenkins jobs we have.  I'd just use `def` everywhere.  (Or change everyone to use the return type in all our jobs, but I don't think you want to take that on.)